src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java

```diff
@@ -6,17 +6,15 @@ public interface IndexerIO {
 
     @AutoLog
     class IndexerIOInputs {
-        public double intakeVoltage;
-        public double intakeCurrentAmps;
-        public double intakeVelocityRPM;
-        public double intakeTempCelsius;
-        public double feederVoltage;
-        public double feederCurrentAmps;
-        public double feederVelocityRPM;
-        public double feederTempCelsius;
+        public double intakeVoltage = 0.0;
```

Comment:
don't doubles get initialized to 0 by default?

Reply:
yeah i'm just doing this to make this have more consistent formatting with the rest of the IOs, and it's nice to see the default values.